packages/trader/src/Modules/Trading/Components/Elements/purchase-fieldset.jsx

```diff
@@ -10,162 +10,158 @@ import ContractInfo from 'Modules/Trading/Components/Form/Purchase/contract-info
-                            onMouseEnter={() => {
-                                if (!is_disabled) {
-                                    onHoverPurchase(true, type);
-                                }
-                            }}
-                            onMouseLeave={() => {
-                                if (!is_disabled) {
-                                    onHoverPurchase(false);
-                                }
-                            }}
-                        >
-                            <div className='btn-purchase__box-shadow' />
-                            {is_proposal_error && !is_market_closed ? (
-                                <Popover
-                                    has_error
-                                    alignment='left'
-                                    message={info.message}
-                                    is_open={is_proposal_error && !is_market_closed}
-                                    relative_render
-                                    margin={6}
-                                >
-                                    {purchase_button}
-                                </Popover>
-                            ) : (
-                                <React.Fragment>
-                                    {is_multiplier ? (
-                                        <Popover
-                                            alignment='left'
-                                            is_bubble_hover_enabled
-                                            margin={8}
-                                            message={info.message}
-                                            relative_render
-                                        >
-                                            {purchase_button}
-                                        </Popover>
-                                    ) : (
-                                        purchase_button
-                                    )}
-                                </React.Fragment>
-                            )}
-                            {
-                                // is_purchase_confirm_on ?
-                                //     <PopConfirm
-                                //         alignment='left'
-                                //         cancel_text={localize('Cancel')}
-                                //         confirm_text={localize('Purchase')}
-                                //         message={localize('Are you sure you want to purchase this contract?')}
-                                //     >
-                                //         {purchase_button}
-                                //     </PopConfirm>
-                                //     :
-                                //     purchase_button
-                            }
-                        </div>
+                        {
+                            // is_purchase_confirm_on ?
+                            //     <PopConfirm
```

Comment:
Here as well 😄

Reply:
Will do! I was not sure why this block was commented out, so I was not too sure if I should remove it.